### PR TITLE
Slack command: update translations

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add endpoint to respond to Slack command `/update-translations` which triggers the workflow on GH with the same name.

## Justify why they are needed

So Peter can trigger updated translations (PR) from Slack

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2688** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2688" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2688?utm_source=stack-comment).

File: .github/workflows/download-translations.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch:
 
 permissions:
-  contents: read
+  contents: write
```
Comment (robinandeer) [inline .github/workflows/download-translations.yml:7]: Needed to create pull request

File: .github/workflows/download-translations.yml [hunk 2]
```diff
@@ -47,5 +47,4 @@ jobs:
             This PR was created automatically by the "Download translations from Lokalise" action.
           branch: copy/update-translations/${{ github.run_id }}
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (robinandeer) [inline .github/workflows/download-translations.yml:50]: It's already the default

File: apps/store/src/pages/api/slack/update-translations.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  if (req.query.secret !== process.env.SLACK_SECRET) {
+    return res.status(401).json({ message: 'Invalid token' })
+  }
+
+  const { command, response_url, user_id } = req.body as SlashCommand
+
+  if (command !== '/update-translations') {
+    return res.status(400).json({ message: `Unknown slash command: ${command}` })
```
Comment (guilhermespopolin) [inline apps/store/src/pages/api/slack/update-translations.ts:22]: Do you mean "slack" command? 🤔

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/pages/api/slack/update-translations.ts (hunk 3 above)
Comment (robinandeer) [reply apps/store/src/pages/api/slack/update-translations.ts:22]: Slack has a feature called "slash commands" -- I know it's confusing 😅

**Pull request merged**